.github/workflows/ci.yml

```diff
@@ -1,20 +1,36 @@
-name: Project Tests
+name: Continuous Integration
 on:
   push:
     branches:
-      - master
       - develop
+      - master
   pull_request:
     branches:
-      - master
       - develop
+      - master
 
 jobs:
-  test:
+  build-test-push:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Build Docker image
-        run: docker build -t clerk:latest -f docker/Dockerfile .
-      - name: Run tests
-        run: docker-compose -f docker/docker-compose.local.yml run --rm -e DJANGO_SETTINGS_MODULE=clerk.settings.test test  pytest -s -vv
+      - name: Set tag name
```

Comment:
this part is basically saying that 'if we have pushed to develop branch or pull request into develop branch', then set tag to 'staging', otherwise set to 'prod'.